### DISCUSSION TITLE
PDF Generator

### DISCUSSION THREAD

**Discussion by GCITechSC** [code4recovery/12-step-meeting-list#504] [Q&A] [answered]
Hey all,

Still learning here.  I'm loving what TSML does for our website at aacolumbia.org

I have had the PDF generator, but pretty sure I am underutilizing it.  I STILL haven't made a child theme (scares me a little). 

Here's what I'm currently doing.  Meetings get updated, I go to the Generator, generate new list, download, then update another place on our site where the download link is available by replacing the PDF that I've downloaded.

What I think I could do is make the hyperlink on the "Greater Columbia Intergroup Meeting List" link to this?  But do I have the address correct?
www.aacolumbia.org/meetings/wp-content/uploads/meeting_list.pdf

Is that right?  

Thanks for anyone's help!

Scott

**Comment by tech2serve** [comment, marked as answer]
Hey Scott--

The link to uploaded media can change from site-to-site, so I won't be able to tell you if your link is correct. Here is how I get the link to media (which I can then add to a Wordpress page, or menu item):

While logged into your site and on the admin page:
1. Click on Media-->Library
2. Click on the attachment of interest and a details page should appear
3. Click on "Copy URL to Clipboard"

<img width="377" alt="image" src="https://user-images.githubusercontent.com/31904316/130363314-b1db00ae-6142-47d2-aa5b-e4ec35bce7e5.png">

At that point you should have the accurate URL for the attachment.

**Reply by GCITechSC** [reply]
Thanks for this, but that's essentially how I'm doing it I think.  I am trying to make the right link for what it says within the PDF generator plugin, but only gives a partial link.  

Or am I not making sense of both things?

**Reply by tech2serve** [reply]
> Thanks for this, but that's essentially how I'm doing it I think.  I am trying to make the right link for what it says within the PDF generator plugin, but only gives a partial link.  
> 
> 
> 
> Or am I not making sense of both things? 

If you have an issue with the _12 Step Meeting PDF_ plugin, this is not the same as the _12 Step Meeting List_ (or TSML) plugin. Code for Recovery doesn't have anything to do with the first plugin; we control and provide TSML only. For issue with _12 Step Meeting PDF_, you'll need to contact that author.

Regardless, the procedures for posting a link to media, whether it is a PDF, image, movie, etc., are not affected by TSML or the PDF generator plugin. For issues with that process, you'll need find a generic Wordpress support forum, or you may have some luck in TIAA Forum (if you're AA).

**Reply by GCITechSC** [reply]
Thanks for this, I thought I would ask here.  I do understand it's a different author.
